docs/Contributing.adoc

```diff
@@ -28,6 +28,7 @@ These are our contribution guidelines for helping out with the project. Any sugg
 * Owner of existing role or config must be set as reviewer.
 ** If new role or config, owner must be identified.
 * If your Pull Request is not ready for review, open it as link:https://github.blog/2019-02-14-introducing-draft-pull-requests/[Draft] or prefix with `WIP` in the title.
+* AgnosticD being part of the Red Hat Community of Practices, the link:https://redhat-cop.github.io/contrib/[Red Hat CoP Contibution Guidelines] applies.
```

Comment:
Change to 

`AgnosticD is part of the Red Hat Community of Practices; the link:https://redhat-cop.github.io/contrib/[Red Hat CoP Contribution Guidelines] apply.